ray-operator/controllers/ray/utils/util_test.go

```diff
@@ -840,3 +948,257 @@ func TestIsGCSFaultToleranceEnabled(t *testing.T) {
 		})
 	}
 }
+
+func createRayPodSpec(cpu, memory string) corev1.PodSpec {
```

Comment:
Maybe rename to `createPodSpec` instead? This function doesn't have anything related to Ray (e.g. set Ray related env vars).

```suggestion
func createPodSpec(cpu, memory string) corev1.PodSpec {
```